<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MousePicking</class>
 <widget class="QDialog" name="MousePicking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="PhongLightBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>单击位置 → 世界坐标：</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,13">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>单击的鼠标位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="clickPos">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,13">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>像素深度值 Z</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pixelZ">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,13">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>相机到点击像素距离 W</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ViewPos2ClickPosW">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,13">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>裁剪空间坐标：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ClipPos">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,13">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>像素世界坐标：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="WorldPos">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
